Earthfile

```
@@ -71,23 +75,36 @@ BUILD_AND_INSTALL:
 alpine-base:
     ARG --required version
     FROM alpine:$version
-    RUN apk add cmake ninja-is-really-ninja gcc musl-dev
+    RUN apk add cmake ninja-is-really-ninja gcc musl-dev g++ cyrus-sasl-dev
+
+alpine-build-env-base:
+    ARG --required version
+    FROM +alpine-base --version=$version
+    RUN apk add openssl-dev cyrus-sasl-dev snappy-dev ccache
 
+alpine-test-env-base:
+    ARG --required version
+    FROM +alpine-base --version=$version
+    RUN apk add libsasl snappy pkgconfig
```

```suggestion
    RUN apk add libsasl snappy
```

Remove package config dependency? I assume this is no longer needed due to 0c7694670c339aaf82ddaeee999299daf420ccd4. Comment applies to other package config dependencies in this file: `pkgconf` and `pkg-config`